com.jetbrains.PyCharm-Professional.yaml

```diff
@@ -27,13 +27,14 @@ modules:
       - -DFREEGLUT_BUILD_DEMOS=OFF
     sources:
       - type: archive
-        url: "https://prdownloads.sourceforge.net/freeglut/freeglut-3.2.1.tar.gz"
+        url: https://prdownloads.sourceforge.net/freeglut/freeglut-3.2.1.tar.gz
         sha256: d4000e02102acaf259998c870e25214739d1f16f67f99cb35e4f46841399da68
 
   - name: python3-virtualenv
     buildsystem: simple
     build-commands:
-      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} virtualenv
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+        virtualenv
```

Comment:
@barthalion / @wjt I'm not sure about that. Is this the recommended way of writing it?

Reply:
It's how the used yaml lib dump the file. So it should at least cause no trouble.

Reply:
Yep, build passed. But I feel like this is kind of a line wrapping "issue".

Reply:
Patches welcome to improve the formatting preservation – I did try quite hard but it's surprisingly difficult to do!